### PR TITLE
Add support for suffix and environment variables

### PR DESCRIPTION
**Pull request by menathan**
Implements #72 & #78 
Added few improvements re to contributions and local setup.

### CODE REVIEW

**Comment by menathan** [comment]
@shiouen this one is ready for review

File: main.nu [hunk 1]
```diff
@@ -39,7 +39,7 @@ def get-ldflags [version: string] {
     const go_import_path = "github.com/restechnica/semverbot"
 
     let ldflags = [
-        $"\"-X ($go_import_path)/internal/ldflags.Version=($version)\""
+        $"-X ($go_import_path)/internal/ldflags.Version=($version)"
```
Comment (shiouen) [inline main.nu:42]: This might have to stay as-is, has this been tested on windows?

File: .github/workflows/development.yml [hunk 2]
```diff
@@ -40,14 +40,14 @@ jobs:
       - name: test
         run: make test
 
-      - name: build prerelease
+      - name: build-prerelease
```
Comment (shiouen) [inline .github/workflows/development.yml:43]: to keep styling consistent, could you remove the hyphen again?

File: .github/workflows/development.yml [hunk 3]
```diff
@@ -40,14 +40,14 @@ jobs:
...
         run: |
-          nu main.nu build
+          make build
           sbot version
           sbot update version --debug
 
-      - name: build
+      - name: build-all
```
Comment (shiouen) [inline .github/workflows/development.yml:49]: to keep styling consistent, could you remove the hyphen again?

File: pkg/semver/parse_test.go [hunk 4]
```diff
@@ -14,8 +14,9 @@ func TestParse(t *testing.T) {
 		Major    string
 		Minor    string
 		Patch    string
-		Prebuild string
 		Prefix   string
+		Suffix   string
+		Prebuild string
```
Comment (shiouen) [inline pkg/semver/parse_test.go:19]: Could you sort these alphabetically? Prebuild before Prefix

**Review by shiouen — CHANGES_REQUESTED** [review]
Left some small styling remarks and one remark about a potential issue in regards to Windows. Really appreciate the effort you put into this PR!

File: main.nu (hunk 1 above)
Comment (menathan) [reply main.nu:42]: Haven't tested this on windows, but when I run a `make build` on Mac, I get follow error message:

```sh
make build
building local binary...
invalid value "\"-X github.com/restechnica/semverbot/internal/ldflags.Version=dev\"" for flag -ldflags: parameter may not start with quote character "
```

**Review by shiouen — APPROVED** [review]
lgtm

File: main.nu (hunk 1 above)
Comment (shiouen) [reply main.nu]: hmm alright, let's see if the pipeline runs - I will fix it if it fails

**Pull request merged**